src/utils/validation.ts

```diff
@@ -196,7 +192,7 @@ export function validateISignedRelayerMakerOrder(
 }
 
 export function validateINewOrderSchema(order: INewOrder) {
-  if (typeof order.expiry !== "number" || order.expiry < 0) {
+  if (!Number.isFinite(order.expiry) || order.expiry < 0) {
```

Comment:
perhaps isInteger?

Reply:
Thanks for this. Yep `Number.isInteger` sounds good.